tests/unit/models/builder-test.js

```diff
@@ -181,6 +179,21 @@ describe('models/builder.js', function() {
     });
   });
 
+  describe('cleanup', function() {
+    beforeEach(function() {
+      builder = new Builder({
+        setupBroccoliBuilder,
+        project: new MockProject(),
+        processBuildResult(buildResults) { return Promise.resolve(buildResults); },
+      });
+    });
+
+    it('is idempotent', function() {
```

Comment:
Minor nit-pick: Can we return the promise from the test so it can complete and avoid leaking?